README.rst

```diff
@@ -31,6 +31,14 @@ repository provides Cartographer SLAM for the `Toyota HSR`_ via
 .. _SLAM: https://en.wikipedia.org/wiki/Simultaneous_localization_and_mapping
 .. _Toyota HSR: http://www.toyota-global.com/innovation/partner_robot/family_2.html
 
+Documentation
+=============
+
+You will find complete documentation for using Cartographer with ROS at `our
+Read the Docs site`_.
+
+.. _our Read the Docs site: https://google-cartographer-ros-for-the-toyota-hsr.readthedocs.io/
```

Comment:
Unfortunately, the directions on this site won't work for HSR :(

1. rosdep is, unfortunately, not setup for HSR binaries at this time.
2. The way that you describe merging the rosinstall into the workspace doesn't match current ROS standards: http://wiki.ros.org/wstool where they suggest you init your ws in the src folder and not in the main directory.  Since this is a ROS package it makes more sense to follow ROS conventions here.